aether-kernel/aether/kernel/api/models.py

```diff
@@ -122,43 +93,13 @@ def __str__(self):
     class Meta:
         app_label = 'kernel'
         default_related_name = 'projects'
-        ordering = ('-modified',)
+        ordering = ['-modified']
+        indexes = [
+            models.Index(fields=['-modified']),
+        ]
```

Comment:
Could you add a comment about approximately how much this improved performance for your particular usecase? Either in the PR description or the commit itself. Would be helpful to have for future readers.